From 0e63ca8fd99807b51252db6d8e91e0f5219133f4 Mon Sep 17 00:00:00 2001
From: Mauro Carvalho Chehab <mchehab@s-opensource.com>
Date: Tue, 6 Mar 2018 04:30:22 -0500
Subject: [PATCH 161/437] media: em28xx: constify a new function

em28xx_duplicate_dev() is static. This were supposed to be
merged on the last patch, but somehow, I forgot "-a" when
I called git commit --amend.

Fixes: be7fd3c3a8c5 ("media: em28xx: Hauppauge DualHD second tuner functionality")
Signed-off-by: Mauro Carvalho Chehab <mchehab@s-opensource.com>
---
 drivers/media/usb/em28xx/em28xx-cards.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/usb/em28xx/em28xx-cards.c b/drivers/media/usb/em28xx/em28xx-cards.c
index 7f5d0b28cb8c..fb42f75a3406 100644
--- a/drivers/media/usb/em28xx/em28xx-cards.c
+++ b/drivers/media/usb/em28xx/em28xx-cards.c
@@ -3435,7 +3435,7 @@ static int em28xx_init_dev(struct em28xx *dev, struct usb_device *udev,
 	return 0;
 }
 
-int em28xx_duplicate_dev(struct em28xx *dev)
+static int em28xx_duplicate_dev(struct em28xx *dev)
 {
 	int nr;
 	struct em28xx *sec_dev = kzalloc(sizeof(*sec_dev), GFP_KERNEL);
-- 
2.16.3

